### PR TITLE
New tutorials in bioimaging x FAIR data management 

### PR DESCRIPTION
**Pull request by wee-snufkin**
Thanks to the collaboration during CoFest GCC2023 with @kkamieniecka and @poterlowicz-lab I got an opportunity to continue the work of @Laura190 and produce two small tutorials focusing on FAIR data management in bioimaging: the first one is more general and the second one goes into more detail, exploring REMBI. 

Other than that, Laura added as a contributor and the new tutorials included as suggested training in the 'Imaging' topic.

### CODE REVIEW

**Comment by bgruening** [comment]
@wee-snufkin do you need help with the failing tests?

File: topics/fair/tutorials/bioimage-REMBI/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+- study
+- study component
+- biosample
+- specimen
+- image acquisition
+- image data
+- image correlation
+- analyzed data
+
+> Within each module, there are attributes that should be included to make the published data FAIR. We will explore all the modules and attributes suggested by REMBI and we'll show some examples as well.
```
Comment (shiltemann) [inline topics/fair/tutorials/bioimage-REMBI/tutorial.md:86]: (getting rid of the `>` at the start of the line should fix the errors, same for most of the other errors below)

**Comment by wee-snufkin** [comment]
> @wee-snufkin do you need help with the failing tests?

Thanks @bgruening, @shiltemann came in the right moment and now all errors are fixed!

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks a lot @wee-snufkin!

@kkamieniecka as a member of the editorial board for this topic, please review this and feel free to merge it when you are happy! It looks good to go from a technical point of view :+1:

**Pull request merged**